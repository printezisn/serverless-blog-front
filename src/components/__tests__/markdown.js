import React from "react"
import renderer from "react-test-renderer"

import Markdown from "../markdown"

describe("Markdown", () => {
    it("renders correctly", () => {
        const text =
            "### Header\n\n" +
            "* List\n" +
            " * Top 1\n" +
            " * Top 2\n" +
            " * Top 3\n"

        const markdown = renderer
            .create(<Markdown text={text}></Markdown>)
            .toJSON()

        expect(markdown).toMatchSnapshot()
    })
})
