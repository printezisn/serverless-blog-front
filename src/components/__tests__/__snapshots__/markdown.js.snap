// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Markdown renders correctly 1`] = `
<div
  className="markdown"
  dangerouslySetInnerHTML={
    Object {
      "__html": "<h3 class=\\"title is-3\\">Header</h3><ul>
<li>List<ul>
<li>Top 1</li>
<li>Top 2</li>
<li>Top 3</li>
</ul>
</li>
</ul>
",
    }
  }
/>
`;
